ManatiCR/FDI
# FDI-13: Create derecho content type.

[FDI-13](https://manati.atlassian.net/browse/FDI-13)
-
**Steps to test**
- Run `ahoy drush cim -y`
- Log in as an admministrador de contenido user
- Create one node derecho, check you can edit it and delete it as well
